### PR TITLE
feat: bookmarklet to load config.json

### LINKED ISSUE #143: [mv3] Bookmarklet to load config.json
The bookmarklet (`app.js`) should load `/tools/sidekick/config.json` instead of `/tools/sidekick/config.js` to be in sync with the mv3 extension.

### PR DESCRIPTION
**Pull request by rofe**
Fix #143

### CODE REVIEW

File: src/bookmarklet/app.js [hunk 1]
```diff
@@ -60,20 +58,25 @@
...
+                ...(await res.json()),
+              };
+            }
+          } catch (e) {
+            console.log('error retrieving custom sidekick config', e);
           }
+          // apply base config
+          window.hlx.sidekickConfig = {
+            ...window.hlx.sidekickConfig,
+            ...baseConfig,
```
Comment (tripodsan) [inline src/bookmarklet/app.js:77]: so the `baseConfig` wins?

**Review by tripodsan — APPROVED** [review]

File: src/bookmarklet/app.js [hunk 2]
```diff
@@ -60,20 +58,25 @@
...
-            document.head.append(configScript);
+          try {
+            const res = await fetch(`${configOrigin}/tools/sidekick/config.json`);
+            if (res.ok) {
+              console.log('custom sidekick config found');
+              // apply custom config
+              window.hlx.sidekickConfig = {
+                ...window.hlx.sidekickConfig,
+                ...(await res.json()),
+              };
```
Comment (tripodsan) [inline src/bookmarklet/app.js:69]:
```suggestion
              Object.apply(window.hlx.sidekickConfig, await res.json());
```

File: src/bookmarklet/app.js (hunk 1 above)
Comment (tripodsan) [inline src/bookmarklet/app.js:77]:
```suggestion
          Object.assign(window.hlx.sidekickConfig, baseConfig);
```

File: src/bookmarklet/app.js [hunk 3]
```diff
@@ -60,20 +58,25 @@
...
-          });
-          // init sidekick via project config
-          if (document.getElementById(configScript.id)) {
-            document.getElementById(configScript.id).replaceWith(configScript);
-          } else {
-            document.head.append(configScript);
+          try {
+            const res = await fetch(`${configOrigin}/tools/sidekick/config.json`);
+            if (res.ok) {
+              console.log('custom sidekick config found');
```
Comment (tripodsan) [inline src/bookmarklet/app.js:64]:
```suggestion
              console.log(`custom sidekick config loaded from ${configOrigin}/tools/sidekick/config.json`);
```

**Review by tripodsan — APPROVED** [review]

File: src/bookmarklet/app.js (hunk 1 above)
Comment (rofe) [reply src/bookmarklet/app.js:77]: Yes: the base config contains critical properties like `owner`, `repo` and `ref`, which don't make sense to overlay in `config.json`.

File: src/bookmarklet/app.js (hunk 2 above)
Comment (rofe) [reply src/bookmarklet/app.js:69]: That's what I had before, but I thought the spread operator notation should be preferred? `eslint` didn't seem to have an opinion either way though...

**Pull request merged**